### PR TITLE
[MM-39661] MM-T827 Select Next/Previous Tab

### PR DESCRIPTION
**Pull request by shadowshot-x**
#### Summary

Added End to End tests for Tab change monitoring.

#### Ticket Link

Fixes https://github.com/mattermost/desktop/issues/1904

#### Checklist

- [x] Added E2E tests
- [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
- [x] executed `npm run lint:js` for proper code formatting

#### Device Information
This PR was tested on:  Mac OS - Big Sur, v11.4

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by devinbinnie — CHANGES_REQUESTED** [review]
@shadowshot-x Thanks for the PR!

Unfortunately this approach is not what is required for this test. You will need to use RobotJS to navigate the application menu (hence why you need to navigate to Window first) and select the tabs from there. There are examples of RobotJS you can use in the codebase.

**Review by devinbinnie — CHANGES_REQUESTED** [review]
Thanks for the PR @shadowshot-x!
I have a comment for you to review.

File: e2e/specs/menu_bar/window_menu.test.js [hunk 1]
```diff
@@ -76,4 +76,19 @@ describe('mattermost', function desc() {
...
         dropdownButtonText.should.equal('example');
     });
+
+    it('MM-T827 select next/previous tab', async () => {
+        const mainView = this.app.windows().find((window) => window.url().includes('index'));
+
+        let tabViewButton = await mainView.innerText('.active');
+        tabViewButton.should.equal('Channels');
+
+        robot.keyTap('2', [process.platform === 'darwin' ? 'command' : 'control']);
```
[the remark this answers is not in the document]
Comment (shadowshot-x) [reply e2e/specs/menu_bar/window_menu.test.js:86]: Absolutely Correct. I will make the changes and push this weekend. Sorry for the late reply, was a bit busy this week

**Comment by devinbinnie** [comment]
@shadowshot-x Are you still around to finish this up?

**Review by devinbinnie — APPROVED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]

**Pull request merged**